kong/conf_loader/init.lua

```diff
@@ -360,6 +360,7 @@ local CONF_PARSERS = {
   pg_keepalive_timeout = { typ = "number" },
   pg_pool_size = { typ = "number" },
   pg_backlog = { typ = "number" },
+  pg_ttl_cleanup_interval = { typ = "number" },
```

Comment:
Make this an explicit hidden flag like `_debug_pg_ttl_cleanup_interval` instead. Otherwise we will have to document it.

Reply:
done